pdftotree/utils/pdf/pdf_utils.py

```diff
@@ -188,6 +196,28 @@ def processor(m):
 
             # Collect stats on the chars
             if isinstance(m, LTChar):
+                # Construct LTTextContainer from LTChar(s) outside of LTTextContainer
+                nonlocal _font
+                nonlocal mention
+                font = (m.fontname, m.size)
+                if font != _font:
+                    if _font is not None:
+                        mention.font_name, mention.font_size = _font_of_mention(mention)
+                        layout_container = LTLayoutContainer((0, 0, 0, 0))  # dummy bbox
+                        laparams = LAParams(
+                            char_margin=1.0, word_margin=0.1, detect_vertical=True
+                        )
```

Comment:
Do not use hard-coded parameters. Use the same `LAParams` in pdf_utils.analyze_pages().

Reply:
Resolved by 4566ff86bf0de846d46aa3df48562d934357f5ec.